src/components/Main.svelte

```diff
@@ -123,26 +135,36 @@
 	<Button class="mx-8 col-span-2" disabled={!file} on:click={resetComponents}>Reset</Button>
 	<Button class="mx-1 col-span-2" disabled={!file} on:click={download}>Download</Button>
 </div>
-{#if toShowAlert}
-	<div class="grid grid-cols-4">
-		<Alert class="mt-1 mb-3">
-			<svelte:component this={alertComponent} message={alertErrorMessage} on:close={closeAlert}/>
-		</Alert>
+<main class="relative">
+	{#if toShowAlert}
+		<div class="grid grid-cols-4">
+			<Alert class="mt-1 mb-3">
+				{#if alertComponent === undefined}
```

Comment:
Ugh, ugly. :disappointed:
Though I want to avoid `{@html }`

Reply:
Created an [issue](https://github.com/Attacktive/witchspring-r-editor/issues/30).